content/posts/2021-07-30-scm-manager-2-22-0/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: SCM-Manager 2.22.0
+date: 2021-07-30T15:00:00+0000
+author: Eduard Heimbuch
+categories:
+- scm-manager
+- release
+keywords:
+- caching
+- search
+- overview
+- redesign
+---
+
+Hey SCM-Manager community,
+
+today we present you the next iteration of the new omni search and also the new design of the repository overview.
+We also added powerful caching for internal working copies and fixed some bugs you reported.
+
+## Omni Search - Details Page
+Before you could search for repositories using the omni search in the header. Now you may still search for repositories shown as quick result, but also you can show `More results`. 
+This leads to the new search details page which shows all search results categorized by result type. If you search for repository descriptions we also added highlighting inside the matching text fragments.
+
+Tell us what you think about the new features and also stay excited since we just started to implement the code search, so you can search inside your repository files.
+
+![Omni Search](assets/details-search.png)
+
+## UI Redesign - Repository Overview
+
+After the header, we have now redesigned the repository overview. We hope it is now clearer and easier to use.
+We really need your opinion on this, so please give us feedback on how you like our changes.
+
+![Repository Overview](assets/repository-overview.png)
+
+## Caching for Working Directories
+
+This feature is still experimental and a little hidden. Now you can activate caching for the internal working copies. 
+This is especially helpful if you work a lot with the [Editor plugin](https://scm-manager.org/plugins/scm-editor-plugin/) since it boosts the performance dramatically.
```

Comment:
```suggestion
This is especially helpful if you work a lot with the [editor plugin](https://scm-manager.org/plugins/scm-editor-plugin/) or the [review plugin](https://scm-manager.org/plugins/scm-review-plugin/), since it boosts the performance dramatically.
```